static/widgets/toggles.ts

```diff
@@ -107,12 +107,15 @@ export class Toggles extends EventEmitter {
         this.emit('change', before, this.get());
     }
 
-    enableToggle(key: string, enable: boolean) {
+    enableToggle(key: string, enable: boolean, lockChecked: boolean = false) {
```

Comment:
I think this function is doing too much, this just for re-enabling a greyed out button. I think it'd be better to add a separate control for setting the toggle, if we don't have one already.

Reply:
I agree. Will fix.